debug-ui/scripts/stats-server.ts

```diff
@@ -1219,14 +1206,40 @@ export class ManifestStatsServer {
       );
 
       for (const row of checkpointResult.rows) {
-        this.baseVolumeAtomsCheckpoints.set(
-          row.market,
-          row.base_volume_checkpoints,
-        );
-        this.quoteVolumeAtomsCheckpoints.set(
-          row.market,
-          row.quote_volume_checkpoints,
-        );
+        // Parse JSON if stored as strings
+        let baseCheckpoints = row.base_volume_checkpoints;
+        let quoteCheckpoints = row.quote_volume_checkpoints;
+
+        // Convert to arrays if needed
+        if (typeof baseCheckpoints === 'string') {
+          baseCheckpoints = JSON.parse(baseCheckpoints);
+        }
+
+        if (typeof quoteCheckpoints === 'string') {
+          quoteCheckpoints = JSON.parse(quoteCheckpoints);
+        }
+
+        // Ensure they are arrays
+        if (!Array.isArray(baseCheckpoints)) {
```

Comment:
if db fails, why resetting? db should just be there for saving and restart, not overriding the current.

Reply:
The db isn't failing afaik. Stats sever is but then we override the 24 hrs volume because savecheckpoint ran before we loadstate on the db

Reply:
So the concern is that we are doing a save of zero on restart and then using that immediately after?
Should have a db load in the initialization then.